configuration/drugs/drugs.csv

```diff
@@ -113,4 +113,9 @@ df88de34-ead9-4b51-9efa-30a4dbc0330b,,Thramadol hydrochloride 100mg/2ml,Cambodia
 4784f9f7-3395-4e06-8b97-53c84f5fd5df,,Dextrose 10% 500ml,Cambodia_Dextrose 10%,Solution,500ml,
 423e6510-7316-47be-9ff5-437ed0a46244,,Dextrose 5% and 1/3 NSS 500ml,Cambodia_Dextrose 5% and 1/3 NSS,Solution,500ml,
 8201a1bb-8906-4e11-8951-0ec3ce31a6d0,,Lactate Ringer (LR) 500ml,Cambodia_Lactate Ringer (LR),Solution,500ml,
-efac339f-b43f-42dc-8c87-5b57d8056678,,Sodium Chloride 0.9% 500ml,Cambodia_Sodium Chloride 0.9%,Solution,500ml,
\ No newline at end of file
+efac339f-b43f-42dc-8c87-5b57d8056678,,Sodium Chloride 0.9% 500ml,Cambodia_Sodium Chloride 0.9%,Solution,500ml,
+,,OK Pill N/A,Cambodia_OK Pill,Tablet,N/A,

```

Comment:
I think it is safer with drugs to always put the UUID (since duplicate names are possible!!).

There is a nice tool on Linux to generate UUIDs for you: `uuidgen`
You can also directly copy the new UUID to your clipboard using the following command (need **xclip** to be installed):
`uuidgen | xclip -sel clip`
After running this command a UUID is saved in your clipboard and simply doing CTRL + V (or Paste) would paste the UUID.

Do you want to try this and amend the commit? Or do you prefer I do it?

Reply:
@rbuisson ok i will do it

Reply:
@rbuisson amended commit to include UUIDs